common/lib/xmodule/xmodule/modulestore/__init__.py

```diff
@@ -602,7 +602,7 @@ def find_asset_metadata(self, asset_key, **kwargs):
 
     @contract(
         course_key='CourseKey', asset_type='None | basestring',
-        start='int | None', maxresults='int | None', sort='tuple(str,(int,>=1,<=2))|None'
+        start='int | None', maxresults='int | None', sort='tuple(str,int) | None'
```

Comment:
Nit: IIRC we're not actually using contracts at this point, so this is probably existing behavior, but does it make sense to put a check in the code that validates the value for sort order? Currently it looks like it will just silently fail to ascending, which might introduce confusion. I'm not sure where this is being used or how obvious the sort params will be there, so feel free to disregard.

Reply:
This asset metadata code is not actually used. I wrote it and it supports storing asset metadata within the modulestore instead of only in GridFS. The project stalled and we never used the functionality. So - valid concern but not worth worrying about at present.